README.md

```diff
@@ -5,24 +5,100 @@ Python bindings for the awesome Openpose library.
 Openpose github page:
 https://github.com/CMU-Perceptual-Computing-Lab/openpose
 
+notice!! PyOpenPose uses OpenCV 3.2 and protobuf 2.6.1
```

Comment:
protobuf is not a requirement of pyopenpose. It is a requirement of openpose so it should not be noted here.
Also opencv 3.x is the requirement since it is known to work with opencv 3.1 as well.